test/OpenFeature.Contrib.Providers.Flagd.Test/FractionalEvaluatorTest.cs

```diff
@@ -55,7 +55,68 @@ public void Evaluate(string email, string flagKey, string expected)
             // Act & Assert
             var result = evaluator.Apply(rule, data);
             Assert.Equal(expected, result.ToString());
+        }
+
+        [Theory]
+        [MemberData(nameof(FractionalEvaluationTestData.FractionalEvaluationContext), MemberType = typeof(FractionalEvaluationTestData))]
+        public void EvaluateUsingRelativeWeights(string email, string flagKey, string expected)
```

Comment:
we should evaluate if it makes sense to move the json tests of the java contrib sdk from https://github.com/open-feature/java-sdk-contrib/tree/main/providers/flagd/src/test/resources/fractional to the harness, and use them as test for all our implementation of the fractionals. maybe we are still missing some data, which is part of the test code, but i think this would allow us to be more stable throughout all the packages. Normally there should be a way in each of our contrib sdks to load json files and to parameterize tests in some sort of way (assumption for dotnet)